spec/request/api/v1/targets/create_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
+
+describe 'POST api/v1/targets/', type: :request do
+  subject do
+    post api_v1_targets_path, params: params, headers: auth_headers, as: :json
+    response
+  end
+
+  let(:user)           { create :user }
+  let(:topic)          { create :topic }
+
+  context 'with valid params' do
+    let(:params) do
+      {
+        target: {
+          title: 'New target',
+          topic_id: topic.id,
+          longitude: -20.111111,
+          latitude: -30.222222,
+          radius: 150
+        }
+      }
+    end
+
+    it 'returns success' do
+      expect(subject).to have_http_status(:success)
+    end
+
+    it 'creates the target' do
+      expect {
+        subject
+      }.to change(Target, :count).by(1)
+    end
+  end
+
+  context 'with invalid params' do
```

Comment:
Maybe you can also add a test case for a request with no user logged in, it should return unauthorized